src/SprykerEco/Zed/Braintree/Business/Payment/Transaction/CaptureItemsTransaction.php

```diff
@@ -207,7 +207,7 @@ protected function addOrderIdToAttributes(array $attributes): array
     protected function getShipmentExpenses(iterable $expenseTransfers): int
     {
         foreach ($expenseTransfers as $expenseTransfer) {
-            if ($expenseTransfer->getType() === ShipmentConstants::SHIPMENT_EXPENSE_TYPE) {
+            if ($expenseTransfer->getType() === ShipmentConfig::SHIPMENT_EXPENSE_TYPE) {
```

Comment:
This is a BC break. See my suggestion for the fix:

```suggestion
            if ($expenseTransfer->getType() === static::SHIPMENT_EXPENSE_TYPE) {
```

... and on top of the class:
```
/**
 * @see \Spryker\Shared\Shipment\ShipmentConfig::SHIPMENT_EXPENSE_TYPE|\Spryker\Shared\Shipment\ShipmentConstants::SHIPMENT_EXPENSE_TYPE
 */
protected const SHIPMENT_EXPENSE_TYPE = 'SHIPMENT_EXPENSE_TYPE';
```

Please fix all occurrences in this PR.